sql/core/src/main/scala/org/apache/spark/sql/streaming/protobuf/StreamingQueryDataSerializer.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.streaming.protobuf
```

Comment:
@gengliangwang Should it be in this package? Please correct me ~ Thanks

Reply:
Shall we put it under the same directory of `SQLExecutionUIDataSerializer`?

Reply:
`StreamingQueryData` is `private[sql]`,  can't access from `org.apache.spark.status.protobuf.sql` package, need change to `private[spark]` like `JobDataWrapper`. Need to change?

Reply:
Yeah it's ok to have such changes